cookbooks/aws-parallelcluster-environment/recipes/install.rb

```diff
@@ -13,6 +13,7 @@
 
 include_recipe "aws-parallelcluster-environment::isolated_install"
 include_recipe "aws-parallelcluster-environment::cfn_bootstrap"
+include_recipe "aws-parallelcluster-environment::cfnconfig_mixed"
```

Comment:
This is an issue. With this change we are moving the recipe from a config phase to install phase.